datumaro/plugins/velodynepoints_format/converter.py

```diff
@@ -0,0 +1,335 @@
+import logging as log
```

Comment:
There should be a license header with MIT license

Reply:
Resolved